src/components/Toggle/Toggle.tsx

```diff
@@ -72,12 +72,22 @@ export class Toggle extends React.Component<IToggleProps, IToggleState> {
               role='button'
               aria-pressed={ isChecked }
```

Comment:
Even though the binding of pressed to checked looks right from the code perspective, when I listed to the example in JAWS - after a switch or two it started saying the opposite. It didn't matter if it was cursor mode (++) or not...